test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultVisualStudioRazorParserIntegrationTest.cs

```diff
@@ -14,63 +16,32 @@
 using Microsoft.VisualStudio.Test;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
-using Microsoft.VisualStudio.Text.Operations;
 using Moq;
 using Xunit;
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
-    public class VisualStudioRazorParserTest : ForegroundDispatcherTestBase
+    public class DefaultVisualStudioRazorParserIntegrationTest : ForegroundDispatcherTestBase
     {
         private const string TestLinePragmaFileName = "C:\\This\\Path\\Is\\Just\\For\\Line\\Pragmas.cshtml";
+        private const string TestProjectPath = "C:\\This\\Path\\Is\\Just\\For\\Project.csproj";
 
-        [Fact]
-        public void ConstructorRequiresNonNullPhysicalPath()
-        {
-            Assert.Throws<ArgumentException>("filePath", 
-                () => new VisualStudioRazorParser(
-                    Dispatcher, 
-                    new TestTextBuffer(null), 
-                    CreateTemplateEngine(), 
-                    null,
-                    new DefaultErrorReporter(),
-                    new TestCompletionBroker(), 
-                    new Mock<VisualStudioDocumentTrackerFactory>().Object,
-                    new Mock<IEditorOperationsFactoryService>().Object));
-        }
-
-        [Fact]
-        public void ConstructorRequiresNonEmptyPhysicalPath()
-        {
-            Assert.Throws<ArgumentException>("filePath", 
-                () => new VisualStudioRazorParser(
-                    Dispatcher, 
-                    new TestTextBuffer(null), 
-                    CreateTemplateEngine(), 
-                    string.Empty, 
-                    new DefaultErrorReporter(),
-                    new TestCompletionBroker(),
-                    new Mock<VisualStudioDocumentTrackerFactory>().Object,
-                    new Mock<IEditorOperationsFactoryService>().Object));
-        }
-
-        // [Fact] Silent skip to avoid warnings. Skipping until we can control the parser more directly.
+        [ForegroundFact]
         private void BufferChangeStartsFullReparseIfChangeOverlapsMultipleSpans()
```

Comment:
Is this passing? We don't want 'silent skips' We want visible skips with bugs associated.

Reply:
Yup, somehow it's running private tests lol; totally missed it. Will make public.